docs/how-to/publish-slice.rst

```diff
@@ -0,0 +1,10 @@
+How to make my slice definitions available for everyone?
+********************************************************
+
+So now, you have created and tested your slice and want to make it **available for everyone.**
```

Comment:
Could you please elaborate a bit more? This does tell people what to do, but with very little context.

Reply:
Done